### DISCUSSION TITLE
Forked Template 404

### DISCUSSION THREAD

**Discussion by healthcalc** [community/community#23858] [Other Feature Feedback, Questions, & Ideas]
<p>I utilized <a href="https://github.com/emilbaehr/automatic-app-landing-page" rel="noopener nofollow ugc">this template </a>, I followed the instructions but when I try to access <a href="https://healthcalc.github.io/healthcalcapp" rel="noopener nofollow ugc">https://healthcalc.github.io/healthcalcapp</a> I get a 404 error?</p>

**Comment by yoannchaudet** [comment]
<p>Hello and welcome here!</p>
<p>Your repository just does not have Pages enabled yet. You can do that in the settings of your repositories under the Pages section.</p>
<p>You can find detailed instructions for that right there: <a href="https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site" class="inline-onebox">Configuring a publishing source for your GitHub Pages site - GitHub Docs</a>.</p>
<p>Hope that helps,</p>
